servers/rendering/renderer_rd/renderer_scene_render_rd.cpp

```diff
@@ -2086,7 +2086,7 @@ void RendererSceneRenderRD::_render_buffers_post_process_and_tonemap(const Rende
 	ERR_FAIL_COND(!rb);
 
 	RendererSceneEnvironmentRD *env = environment_owner.get_or_null(p_render_data->environment);
-	//glow (if enabled)
+	// glow and override exposure (if enabled)
```

Comment:
While you're at it 😄
```suggestion
	// Glow and override exposure (if enabled).
```
https://docs.godotengine.org/en/stable/community/contributing/code_style_guidelines.html#comment-style-guide